### PR TITLE
Meowing squirell

### LINKED ISSUE #1075: Application startup modal
The step between _Welcome screen_ and _Onboarding_ with analytics opt-in.

**Zeplin** [here](https://app.zeplin.io/project/5dadb7820bdfd3832e04afca/screen/5dfa1af7734c3d9725eb2300)

### LINKED ISSUE #1243: Wipe device dialog
**[Zeplin](https://app.zeplin.io/project/5dadb7820bdfd3832e04afca/screen/5e4fcf5f79f38961aebe0e8f)**

Add confirmation & detail explanation of the action before the actual wipe.

### LINKED ISSUE #1486: Button misaligned
Button is not centered. And maybe we need some texting here? @matejzak 

image

- [ ] also check if auto webusb disable for pre-webusb devices works

### LINKED ISSUE #1494: App state modals
In modals showing that there is something wrong with the device and user can't just proceed, we need to have these things:

- [ ] Design
- [ ] Select another connected device if possible (trigger device menu)
- [ ] Have a "resolve button"

### LINKED ISSUE #1511: device switch - too many devices
What I have connected:
T1 in bootloader mode
TT in normal mode:

what I see: 

image

### PR DESCRIPTION
**Pull request by mroz22**
each commit contains a nitpick visual change

___
close #1243 
close #1486 
close #1075
close #1494 
close #1511

### CODE REVIEW

File: packages/suite/src/actions/suite/trezorConnectActions.ts [hunk 1]
```diff
@@ -62,8 +70,8 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
             process.env.SUITE_TYPE === 'desktop'
                 ? resolveStaticPath('connect/')
                 : // : 'https://connect.trezor.io/8/';
-                  //   'https://localhost:8088/';
-                  'https://connect.corp.sldev.cz/develop/';
+                  'https://localhost:8088/';
```
Comment (szymonlesisz) [inline packages/suite/src/actions/suite/trezorConnectActions.ts]: ?

File: packages/suite/src/actions/suite/trezorConnectActions.ts [hunk 2]
```diff
@@ -14,6 +14,14 @@ export const init = () => async (dispatch: Dispatch, getState: GetState) => {
...
         // dispatch event as action
         delete event.event;
+        // console.warn(event);
+        // const random = 'Random' + Date.now();
+        // if (event.payload.id && event.payload.mode === 'initialize' && event.payload.features.device_id) {
+        //     console.log('now we are mocking everchanging device id')
+        //     event.payload.id = random;
+        //     event.payload.features.device_id = random;
+        // }
+
```
Comment (szymonlesisz) [inline packages/suite/src/actions/suite/trezorConnectActions.ts]: ?

File: packages/suite/src/actions/suite/trezorConnectActions.ts (hunk 1 above)
Comment (mroz22) [reply packages/suite/src/actions/suite/trezorConnectActions.ts]: sry, wrongly typed `git add` it just slipped through

File: yarn.lock [hunk 3]
```diff
@@ -8239,9 +8239,9 @@ caseless@~0.12.0:
   integrity sha1-G2gcIf+EAzyCZUMJBolCDRhxUdw=
 
 cashaddrjs@^0.3.10, cashaddrjs@^0.3.3:
```
Comment (mroz22) [inline yarn.lock]: reset

File: packages/suite/src/views/onboarding/steps/ResetDevice/index.tsx [hunk 4]
```diff
@@ -25,10 +26,13 @@ const ResetDeviceStep = (props: Props) => {
             </Wrapper.StepHeading>
             <Wrapper.StepBody>
                 {!isShamirBackupAvailable() && (
-                    <Text>
-                        You are one step from creating your wallet. By clicking the button below you
-                        agree with <Link href={TOS_URL}>TOS</Link>
-                    </Text>
+                    <>
+                        <Text>
+                            You are one step from creating your wallet. By clicking the button below
```
Comment (slowbackspace) [inline packages/suite/src/views/onboarding/steps/ResetDevice/index.tsx:31]: l10n missing nitpick :D

**Pull request merged**